### DISCUSSION TITLE
Seb server - problem starting exam

### DISCUSSION THREAD

**Discussion by Rapacioli** [SafeExamBrowser/seb-win-refactoring#912] [Q&A]
Hello I hope you are well.
I need help.
I have two students, who use Windows, they installed the latest version of SEB. When they download the connection file to enter the exam, this screen appears, from which they cannot exit:
![Seb problem](https://github.com/SafeExamBrowser/seb-server/assets/94466010/d2272d9c-edfc-48a8-9ef0-d8561be39fc1)

The file appears as seb file, not as seb configuration file.

Can you please help me?

Thank you in advance

Melina Rapacioli

**Comment by anhefti** [comment]
Hi,

.seb Files should open the SEB client, not the installer of SEB as it seems to be the case in this screenshot. But ".seb" extension is correct and is for SEB Configuration Files. Most probably, this is a problem with Application File association on the Windows device. Try to right-click on the File and choose "Open With" then choose Safe Exam Browser to open the. 

![image](https://github.com/SafeExamBrowser/seb-server/assets/39078777/7479ac32-9ef3-47f0-a0da-2e6237fadec9)

Set this also as the default to open ".seb" files.

**Reply by Rapacioli** [reply]
Thank you very much for your prompt response. Yes, I did that, and it kept appearing the same. 
I even asked the student to uninstall and reinstall seb but he kept making that mistake.

**Reply by Rapacioli** [reply]
Maybe I should ask students not to install the seb in the downloads folder...

**Reply by anhefti** [reply]
Usually it should not depend on from where the installer is started, it should install SEB on the right place. Does the Installation itself goes through without errors?

Please also note that this is a SEB Server forum. Please ask your question about SEB Client Installation on a [SEB discussion forum](https://github.com/SafeExamBrowser/seb-win-refactoring) for proper answer.
